conf/agent.conf.template

```diff
@@ -73,8 +73,6 @@ cygnusagent.sinks.hdfs-sink.hdfs_port = 14000
 cygnusagent.sinks.hdfs-sink.hdfs_username = hdfs_username
 # password for the username
 cygnusagent.sinks.hdfs-sink.hdfs_password = xxxxxxxxxxxxx
-# HDFS backend type (webhdfs, httpfs or infinity)
```

Comment:
It was never implemented the interface with Infinity, thus it was never correct to have Infinity there.